### PR TITLE
Clarifying what is not supported for PostgreSQL

### PR DESCRIPTION
**Pull request by coltoneshaw**
It seems we will discontinue support for all PostgreSQL 9.x and below versions. Some customers might read this as 9.5 / 9.6 is supported. I clarified the language slightly for this.

### CODE REVIEW

File: source/administration/important-upgrade-notes.rst [hunk 1]
```diff
@@ -2,7 +2,7 @@ Important Upgrade Notes
...
+   PostgreSQL ended long-term support for `version 9.4 in February 2020 <https://www.postgresql.org/support/versioning>`_. Mattermost is officially supporting PostgreSQL version 10 with v5.26 release as PostgreSQL 9.4 is no longer supported. New installs will require PostgreSQL 10+. Previous Mattermost versions, including our current ESR, will continue to be compatible with PostgreSQL 9.4. We plan on fully deprecating PostgreSQL 9.4, and all 9.x releases in our v5.30 release (December 16). Please follow the instructions under the Upgrading Section within `the PostgreSQL documentation <https://www.postgresql.org/support/versioning/>`_.
```
Comment (amyblais) [inline source/administration/important-upgrade-notes.rst:5]:
```suggestion
   PostgreSQL ended long-term support for `version 9.4 in February 2020 <https://www.postgresql.org/support/versioning>`_. Mattermost is officially supporting PostgreSQL version 10 with v5.26 release as PostgreSQL 9.4 is no longer supported. New installs will require PostgreSQL 10+. Previous Mattermost versions, including our current ESR, will continue to be compatible with PostgreSQL 9.4. We plan on fully deprecating PostgreSQL 9.4 and all 9.x versions in our v5.30 release (December 16). Please follow the instructions under the Upgrading Section within `the PostgreSQL documentation <https://www.postgresql.org/support/versioning/>`_.
```

**Review by amyblais — COMMENTED** [review]
Mind also updating the text in the changelog https://docs.mattermost.com/administration/changelog.html#compatibility, or let me know if I can help with this.

File: source/administration/important-upgrade-notes.rst (hunk 1 above)
Comment (coltoneshaw) [reply source/administration/important-upgrade-notes.rst:5]: Just updated the changelog. Let me know if I did it right!

**Review by amyblais — APPROVED** [review]
Thanks, looks good, I committed my suggestion and fixed one typo.

**Pull request merged**